website/docs/modules/filter_genotypes.md

```diff
@@ -0,0 +1,173 @@
+* Reference context with respect to UCSC Genome Browser tracks:
+    * RepeatMasker
+    * Segmental duplications
+    * Simple repeats
+    * K-mer mappability (umap_s100 and umap_s24)
+
+### Model availability
+
+For ease of use, we provide a model pre-trained on high-quality data with truth data derived from long-read calls:
+```
+gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/gatk-sv-recalibrator.aou_phase_1.v1.model
+```
+See the SV "Genotype Filter" section on page 34 of the [All of Us Genomic Quality Report C2022Q4R9 CDR v7](https://support.researchallofus.org/hc/en-us/articles/4617899955092-All-of-Us-Genomic-Quality-Report-ARCHIVED-C2022Q4R9-CDR-v7) for further details on model training.
+
+### SL scores
+
+All valid genotypes are annotated with a "scaled logit" (`SL`) score, which is rescaled to non-negative adjusted `GQ` values on [1, 99]. Note that the rescaled `GQ` values should *not* be interpreted as probabilities. Original genotype qualities are retained in the `OGQ` field.
+
+A more positive `SL` score indicates higher probability that the given genotype is not homozygous for the reference allele. Genotypes are therefore filtered using `SL` thresholds that depend on SV type and size. This workflow also generates QC plots using the [MainVcfQc](https://github.com/broadinstitute/gatk-sv/blob/main/wdl/MainVcfQc.wdl) workflow to review call set quality (see below for recommended practices).
+
+### Modes
+
+This workflow can be run in one of two modes:
+
+1. (Recommended) The user explicitly provides a set of `SL` cutoffs through the `sl_filter_args` parameter, e.g.
+    ```
+    "--small-del-threshold 93 --medium-del-threshold 150 --small-dup-threshold -51 --medium-dup-threshold -4 --ins-threshold -13 --inv-threshold -19"
+    ```
+   Genotypes with `SL` scores less than the cutoffs are set to no-call (`./.`). The above values were taken directly from Appendix N of the [All of Us Genomic Quality Report C2022Q4R9 CDR v7 ](https://support.researchallofus.org/hc/en-us/articles/4617899955092-All-of-Us-Genomic-Quality-Report-ARCHIVED-C2022Q4R9-CDR-v7). Users should adjust the thresholds depending on data quality and desired accuracy. Please see the arguments in [this script](https://github.com/broadinstitute/gatk-sv/blob/main/src/sv-pipeline/scripts/apply_sl_filter.py) for all available options.
+
+2. (Advanced) The user provides truth labels for a subset of non-reference calls, and `SL` cutoffs are automatically optimized. These truth labels should be provided as a json file in the following format:
+    ```
+    {
+      "sample_1": {
+        "good_variant_ids": ["variant_1", "variant_3"], 
+        "bad_variant_ids": ["variant_5", "variant_10"]
+      },
+      "sample_2": {
+        "good_variant_ids": ["variant_2", "variant_13"], 
+        "bad_variant_ids": ["variant_8", "variant_11"]
+      }
+    }
+    ```
```

Comment:
```suggestion
    ```json
    {
      "sample_1": 
      {
        "good_variant_ids": ["variant_1", "variant_3"], 
        "bad_variant_ids": ["variant_5", "variant_10"]
      },
      "sample_2": 
      {
        "good_variant_ids": ["variant_2", "variant_13"], 
        "bad_variant_ids": ["variant_8", "variant_11"]
      }
    }
    ```
```

Reply:
This is very cool